src/zenml/integrations/airflow/orchestrators/airflow_orchestrator.py

```diff
@@ -96,159 +79,323 @@ def __init__(self, **values: Any):
         super().__init__(**values)
         self.airflow_home = os.path.join(
             io_utils.get_global_config_directory(),
-            AIRFLOW_ROOT_DIR,
+            "airflow",
```

Comment:
Is there a specifc reason we aren't using a constant for this anymore?

Reply:
Yeah, I didn't like it :P

No it was in-line with how we handled it in the Kubeflow orchestrator, and it's not something that is used in other places. Additionally, it prevents some random "hacks" like modifying the constant
```python
from zenml.integrations.airflow.orchestrators import airflow_orchestrator
airflow_orchestrator.AIRFLOW_ROOT_DIR = "i mess things up"
```